index.js

```diff
@@ -65,6 +81,13 @@ class Replicate {
       },
     };
 
+    this.files = {
+      create: files.create.bind(this),
+      list: files.list.bind(this),
+      get: files.get.bind(this),
+      delete: files.delete.bind(this),
+    };
```

Comment:
Yeah, for something automatic like this, a minimal API surface area would be ideal. But I think we have enough customer data to suggest that manual control over deleting files is a must-have (like the support request @Pwntus shared just this morning). And if we have that, we need get and list to round out our CRUD operations.

So as much as I'd like to be parsimonious, I'd very much prefer making data management self-serve instead of a manual process.